tasmota/tasmota_xsns_sensor/xsns_05_ds18x20.ino

```diff
@@ -434,7 +446,15 @@ void Ds18x20Name(uint8_t sensor) {
     }
     snprintf_P(DS18X20Data.name, sizeof(DS18X20Data.name), PSTR("%s%c%s"), DS18X20Data.name, IndexSeparator(), address);
 #else
+#ifdef DS18x20_USE_ID_ALIAS
+    if (ds18x20_sensor[sensor].alias) {
+      snprintf_P(DS18X20Data.name, sizeof(DS18X20Data.name), PSTR("DS18Alias%c%d"), IndexSeparator(), ds18x20_sensor[sensor].alias);
```

Comment:
Why not keep the existing standard syntax ? name+separator+index ?

Reply:
DS18X20Data.name not used for sensor type independence.